src/dragula-and-drop.js

```diff
@@ -1,9 +1,10 @@
-import {customElement, bindable, inlineView} from 'aurelia-templating';
+import {customElement, bindable, noView} from 'aurelia-templating';
 import {bindingMode} from 'aurelia-binding';
 import {inject} from 'aurelia-dependency-injection';
 
 import {Options, GLOBAL_OPTIONS} from './options';
 import {Dragula} from './dragula';
+import './dragula.css';
```

Comment:
This is definitely not a safe thing to do in a library!
Sure, with webpack you can set up a `style-loader` and it'll inject the CSS in the page when this module is loaded.

But I don't think it's safe to assume this of all build systems.